src/sections/perps/perpetuals.tsx

```diff
@@ -0,0 +1,190 @@
+import { useState } from 'react';
+import { Flex, FlexProps, Text, Button, Link } from '@chakra-ui/react';
+import { PerpetualSynth } from 'pages/perps';
+import FutureSynthCard from 'src/components/FutureSynthCard';
+
+interface PerpetualsProps extends FlexProps {
+	synths: PerpetualSynth[];
+}
+
+const perpV2Synths = ['sETH'];
```

Comment:
Maybe add a todo saying we need to figure out what futures are live in a better way

Reply:
Yup done